### PR TITLE
Add Hosted regions to resource reference

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/1659

### CODE REVIEW

File: astro/resource-reference-aws.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Read the following document for a reference of our default resources as well as
 
 ## Default cluster values
 
-| Resource                                                                                            | Description                                                                                                                                                                                                                                                                                                                                              | Quantity/ Default Size                       | Configurable? |
+| Resource                                                                                            | Description                                                                                                                                                                                                                                                                                                                                              | Quantity/ Default Size                        | Configurable? |
```
Comment (kwcanuck) [inline astro/resource-reference-aws.md:20]:
```suggestion
| Resource                                                                                            | Description                                                                                                                                                                                                                                                                                                                                              | Quantity/ Default Size                        | Configurable |
```

File: astro/resource-reference-azure.md [hunk 2]
```diff
@@ -38,22 +38,24 @@ To create a new cluster on Astro with a specified configuration, see [Install As
...
+| `japaneast`        | Japan East         | ✔️                            |                |
+| `koreacentral`     | Korea Central      | ✔️                            |                |
+| `northeurope`      | North Europe       | ✔️                            | ✔️              |
+| `southafricanorth` | South Africa North | ✔️                            |                |
+| `southcentralus`   | South Central US   | ✔️                            |                |
+| `uksouth`          | UK South           | ✔️                            |                |
+| `westeurope`       | West Europe        | ✔️                            | ✔️              |
+| `westus3`          | West US 3          | ✔️                            |                |
 
 Modifying the region of an existing Astro cluster isn't supported. If you're interested in a region that isn't on this list, contact [Astronomer support](https://cloud.astronomer.io/support).
```
Comment (kwcanuck) [inline astro/resource-reference-azure.md:60]:
```suggestion
Modifying the region of an existing Astro cluster isn't supported. If you're interested in a region that isn't listed, contact [Astronomer support](https://cloud.astronomer.io/support).
```

File: astro/resource-reference-gcp.md [hunk 3]
```diff
@@ -35,41 +35,43 @@ To create a new cluster on Astro with a specified configuration, see [Create a c
...
+| `us-east1`                | South Carolina, North America | ✔️                            |                |
+| `us-east4`                | Virginia, North America       | ✔️                            | ✔️              |
+| `us-east5`                | Columbus, North America       | ✔️                            |                |
+| `us-south1`               | Dallas, North America         | ✔️                            |                |
+| `us-west1`                | Oregon, North America         | ✔️                            |                |
+| `us-west2`                | Los Angeles, North America    | ✔️                            |                |
+| `us-west3`                | Salt Lake City, North America | ✔️                            |                |
+| `us-west4`                | Nevada, North America         | ✔️                            |                |
 
 Modifying the region of an existing Astro cluster isn't supported. If you're interested in a GCP region that isn't on this list, contact [Astronomer support](https://cloud.astronomer.io/support).
```
Comment (kwcanuck) [inline astro/resource-reference-gcp.md:76]:
```suggestion
Modifying the region of an existing Astro cluster isn't supported. If you're interested in a GCP region that isn't listed, contact [Astronomer support](https://cloud.astronomer.io/support).
```

File: astro/resource-reference-gcp.md [hunk 4]
```diff
@@ -35,41 +35,43 @@ To create a new cluster on Astro with a specified configuration, see [Create a c
...
+| `asia-northeast2`         | Osaka, Asia                   | ✔️                            |                |
+| `asia-northeast3`         | Seoul, Asia                   | ✔️                            |                |
+| `asia-south1`             | Mumbai, Asia                  | ✔️                            |                |
+| `asia-south2`             | Delhi, Asia                   | ✔️                            |                |
+| `asia-southeast1`         | Singapore, Asia               | ✔️                            |                |
+| `asia-southeast2`         | Jakarta, Asia                 | ✔️                            |                |
+| `australia-southeast1`    | Sydney, Australia             | ✔️                            |                |
+| `australia-southeast2`    | Melbourne, Australia          | ✔️                            |                |
+| `europe-central2`         | Warsaw, Europe                | ✔️                            |                |
+| `europe-north1`           | Finalnd, Europe               | ✔️                            |                |
```
Comment (kwcanuck) [inline astro/resource-reference-gcp.md:53]:
```suggestion
| `europe-north1`           | Finland, Europe               | ✔️                            |                |
```

**Review by kwcanuck — APPROVED** [review]

**Comment by chomipi88** [comment]
We have also recently added support for the below GCP instances:

e2-standard-16
e2-highmem-4
e2-highmem-8
e2-highmem-16
e2-highcpu-4
e2-highcpu-8
e2-highcpu-16
n2-standard-4
n2-standard-8
n2-standard-16
n2-highmem-4
n2-highmem-8
n2-highmem-16
n2-highcpu-4
n2-highcpu-8
n2-highcpu-16

File: astro/resource-reference-azure.md [hunk 5]
```diff
@@ -12,14 +12,14 @@ Read the following document for a reference of our default resources as well as
 
 ## Default cluster values
 
-| Resource                                                                                                                 | Description                                                                                                                                                                                                                                                                               | Quantity/ Default Size                                                                         | Configurable? |
+| Resource                                                                                                                 | Description                                                                                                                                                                                                                                                                               | Quantity/ Default Size                                                                        | Configurable? |
```
Comment (jwitz) [inline astro/resource-reference-azure.md:15]:
```suggestion
| Resource                                                                                                                 | Description                                                                                                                                                                                                                                                                               | Quantity/ Default Size                                                                        | Configurable  |
```

File: astro/resource-reference-azure.md [hunk 6]
```diff
@@ -38,22 +38,24 @@ To create a new cluster on Astro with a specified configuration, see [Install As
...
+| `canadacentral`    | Canada Central     | ✔️                            | ✔️              |
+| `centralindia`     | Central India      | ✔️                            |                |
+| `centralus`        | Central US         | ✔️                            |                |
+| `eastasia`         | East Asia          | ✔️                            |                |
+| `eastus`           | East US            | ✔️                            |                |
+| `japaneast`        | Japan East         | ✔️                            |                |
+| `koreacentral`     | Korea Central      | ✔️                            |                |
+| `northeurope`      | North Europe       | ✔️                            | ✔️              |
+| `southafricanorth` | South Africa North | ✔️                            |                |
+| `southcentralus`   | South Central US   | ✔️                            |                |
```
Comment (jwitz) [inline astro/resource-reference-azure.md:55]:
```suggestion
| `southcentralus`   | South Central US   | ✔️                            |                |
| `southeastasia`   | South East Asia   | ✔️                            |    ✔️ (Private Preview)            |
```

File: astro/resource-reference-azure.md [hunk 7]
```diff
@@ -38,22 +38,24 @@ To create a new cluster on Astro with a specified configuration, see [Install As
...
+| `uksouth`          | UK South           | ✔️                            |                |
+| `westeurope`       | West Europe        | ✔️                            | ✔️              |
+| `westus3`          | West US 3          | ✔️                            |                |
```
Comment (jwitz) [inline astro/resource-reference-azure.md:58]:
```suggestion
| `westus2`          | West US 2         | ✔️                            |   ✔️ (Private preview)             |
| `westus3`          | West US 3          | ✔️                            |                |
```

**Pull request merged**